src/Components/Main/Pages/ProfilePage/PostForm/PostFormContainer.js

```diff
@@ -0,0 +1,29 @@
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { PostForm } from './PostForm';
+
+const validationSchema = Yup.object().shape({
+  text: Yup.string()
+    .max(100, 'Must be shorter than 100 characters')
```

Comment:
The same case with the "shorter than" message =)

Reply:
Done!